frontend/src/components/DataCard/datacard.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState, useEffect } from "react";
+import { useInView } from "react-intersection-observer";
+import { FaCity, FaGithub } from "react-icons/fa";
+import { FaUsers } from "react-icons/fa";
+import { RiLandscapeFill } from "react-icons/ri";
+
+// Define props type for StatCard component
+interface StatCardProps {
+  title: string;
+  subText: string;
```

Comment:
```suggestion
  subTitle: string;
```

can also keep it `caption`

Reply:
Point of this please, have retained subText for now.